src/unstract/sdk/adapters/x2text/llm_whisperer/src/static/json_schema.json

```diff
@@ -114,6 +114,18 @@
         "title": "Gaussian Blur Radius",
         "default": 0.0,
         "description": "The radius of the gaussian blur to use for pre-processing the image during OCR based extraction. Useful to eliminate noise from the image. Default is 0.0 if the value is not explicitly set. Available only in the Enterprise version."
+      },
+      "mark_vertical_lines": {
```

Comment:
@jagadeeswaran-zipstack QQ : Is this added to support LLMWhispererv2? Because a new adapter will be added.

Reply:
Not sure. @johnyrahul would be right person to answer this.

Reply:
Yes @harini-venkataraman, I believe we will need this field.